build.sbt

```diff
@@ -136,7 +136,7 @@ lazy val tokenizers = crossProject
   .settings(
     publishableSettings,
     description := "Scalameta APIs for tokenization and their baseline implementation",
-    libraryDependencies += "com.lihaoyi" %%% "fastparse" % "0.4.2",
+    libraryDependencies += "com.lihaoyi" %%% "fastparse" % "0.4.3",
```

Comment:
I opened https://github.com/lihaoyi/Scalatex/pull/52 and published a pre-release of scalatex to our bintray repo with that fix. We can use the pre-release until the official scalatex is out

```scala
// project/plugins.sbt
resolvers += Resolver.bintrayIvyRepo("scalameta", "sbt-plugins") // only needed for scalatex 0.3.8-pre
addSbtPlugin("com.lihaoyi" % "scalatex-sbt-plugin" % "0.3.8-pre")
// build.sbt
resolvers += Resolver.bintrayRepo("scalameta", "maven") // under sharedSettings
```